src/views/pages/name/options-with-welsh.njk

```diff
@@ -0,0 +1,91 @@
+{% set welsh %}
+  {{ govukRadios({
+    errorMessage: errors.name_ending if errors,
+    classes: "govuk-radios",
+    idPrefix: "name_ending",
+    name: "name_ending",
+    items: [
+      {
+        value: "Partneriaeth Cyfyngedig",
+        text: "Partneriaeth Cyfyngedig",
+        id: "nameEnding-1",
+        name: "name_ending",
+        attributes: {
+          required: true
+        }
+      },
+      {
+        value: "PC",
+        text: "PC",
+        id: "nameEnding-2",
+        name: "name_ending",
+        attributes: {
+          required: true
+        }
+      },
+      {
+        value: "P.C.",
+        text: "P.C.",
+        id: "nameEnding-3",
+        name: "name_ending",
+        attributes: {
+          required: true
+        }
+      }
+    ]
+  }) }}
+{% endset -%}
+
+{{ govukRadios({
+  errorMessage: errors.name_ending if errors,
+  classes: "govuk-radios",
+  idPrefix: "name_ending",
+  name: "name_ending",
+  fieldset: {
+    legend: {
+      isPageHeading: false,
+      classes: "govuk-fieldset__legend--xl"
+    }
+  },
+  hint: {
+    text: i18n.namePage.nameEndingHint
+  },
+  items: [
+    {
+      value: "Limited Partnership",
+      text: "Limited Partnership",
+      id: "nameEnding-1",
+      name: "name_ending",
+      attributes: {
+        required: true
+      }
+    },
+    {
+      value: "LP",
+      text: "LP",
+      id: "nameEnding-2",
+      name: "name_ending",
+      attributes: {
+        required: true
+      }
+    },
+    {
+      value: "L.P.",
+      text: "L.P.",
+      id: "nameEnding-3",
+      name: "name_ending",
+      attributes: {
+        required: true
+      }
+    },
+    {
```

Comment:
This is the fly in the ointment. I don't think we can share the option mark-up as this welsh option is in the middle of the gov template code.

Reply:
the items: is a list so you could build the list for english and perhaps and the last item to the list for welsh if needed and have items: myList in the govukRadios. Not sure if that would be any cleaner though.

Reply:
I did try something like this, but including the welsh snippet as an additional option at the end. It didn't like it but will try again.

Reply:
Yes, that worked. Changes pushed.